### DISCUSSION TITLE
blog/2015/2015-05-27-coherence_correlation/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#75] [Blog comments]
# blog/2015/2015-05-27-coherence_correlation/

Note - you can find the nbviewer of this post here A big question that I’ve always wrestled with is the difference between correlation and coherence. Intuitively, I think of these two things as ver...

https://chrisholdgraf.com/blog/2015/2015-05-27-coherence_correlation/

**Comment by cdboschen** [comment]
Very nice write-up Chris! Have you come across "Complex Coherence"?  It is the more general expression for coherence which appears to be mathematically identical to normalized correlation with the frequency time swap (so would then be fully analogous to correlation in the frequency domain).  What is used in this write-up is "Ordinary Coherence" or "Magnitude Squared Coherence". Someday if I have time I'll use your nice templates to compare Complex Cross-Coherence to the Cross-Correlation for further intuition but curious if you've already gone down that path?
